test/helpers/application_helper_test.rb

```diff
@@ -4,9 +4,8 @@ class ApplicationHelperTest < ActionView::TestCase
   include ApplicationHelper
 
   def test_generate_link_for
-    proxy = FactoryBot.create(:dhcp_smart_proxy)
-    subnet = FactoryBot.create(:subnet_ipv4, :name => 'My subnet')
-    proxy.subnets = [subnet]
+    subnet = FactoryBot.build_stubbed(:subnet_ipv4, :name => 'My subnet')
+    proxy = FactoryBot.build_stubbed(:smart_proxy, :dhcp, subnets: [subnet])
```

Comment:
By using `build_stubbed` it saves a database hit. It would be interesting to see how much more we can save. I'd love to have pure unit tests that can run with nulldb. Perhaps it could even use `build`.